### ISSUE TITLE
bug(windows): the button says download the keyboard, but it instead downloads both Keyman and the keyboard in an executable file 

### ISSUE THREAD

**Issue by MakaraSok** [keymanapp/keyman.com#370]

**Comment by darcywong00** [comment]
> ## Reproduce the bug
>No response

This sounds like the case where you don't already have Keyman for Windows installed.

If Keyman for Windows was installed, the download button would just get the kmp file.

**Comment by MakaraSok** [comment]
> 
> This sounds like the case where you don't already have Keyman for Windows installed.
> 
> If Keyman for Windows was installed, the download button would just get the kmp file.

Right, in that case, it may be helpful to users if a note is given as to what is being downloaded so that they can take action accordingly.

Now, even with Keyman installed, the button (the green one on the page) still download the exe. The sentence there though says it should download the keyboard only. :)

<img width="300" alt="image" src="https://github.com/keymanapp/keyman.com/assets/28331388/e0d4e94f-729e-4a81-a867-a5a497c3b43c">

**Comment by mcdurdin** [comment]
Part of the issue here is that end users who are not familiar with Keyman consider the IPA Keyboard to _be_ the thing that they are downloading. The fact that it is a program called Keyman is secondary. So we have to be very careful with the messaging here so that we don't confuse users further.

**Comment by MakaraSok** [comment]
This message is already confusing to me "Your IPA (SIL) keyboard download should start shortly. If it does not,
click the button below to start the download."

**Comment by ermshiperete** [comment]
See also #409
